packages/tree/Tree.scss

```diff
@@ -0,0 +1,5 @@
+@import "~blaze/scss/components.trees";
+
+ul {
+  padding-left: 1em !important;
```

Comment:
where this come from? it should be either part of global css which is currently under PR #1 at least put some comment here. thx